src/main/java/com/kat/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,19 @@
 package com.kat.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+
+    @NotNull
```

Comment:
`@NotNull` allows empty strings (`""`). If the intention is to reject empty titles, use `@NotBlank`:
```suggestion
    @NotBlank(message = "title must not be blank")
```

Reply:
Will change it in another pr, thanks!